nix/mkDevShell.nix

```diff
@@ -22,11 +24,14 @@ let
 
   stableShell = pkgs.mkShellNoCC {
     packages = devDependencies ++ [ bky-as-stable ];
+    shellHook = ''
+      echo "Stable bky-as version: ${version}"
+    '';
```

Comment:
I added these echo statements because when you start the nix shell a few times nothing prints out and it can be easy to forget what version you're running.